pallets/moonbeam-foreign-assets/src/lib.rs

```diff
@@ -207,6 +222,10 @@ pub mod pallet {
 			asset_id: AssetId,
 			xcm_location: Location,
 		},
+		/// Tokens have been locked for asset creation
+		TokensLocked(T::AccountId, AssetId, AssetBalance),
+		/// Lock verification failed
+		LockVerificationFailed(T::AccountId, AssetId),
```

Comment:
can be removed

Reply:
Yes, agreed. I mean the T::Currency::reserve() already emits an Event if I am not mistaken, and this would be enough to check that the deposit was done succesfully.